src/qt/bitcoingui.cpp

```diff
@@ -1211,6 +1213,7 @@ void BitcoinGUI::setHDStatus(bool privkeyDisabled, int hdEnabled)
     labelWalletHDStatusIcon->setToolTip(privkeyDisabled ? tr("Private key <b>disabled</b>") : hdEnabled ? tr("HD key generation is <b>enabled</b>") : tr("HD key generation is <b>disabled</b>"));
 
     // eventually disable the QLabel to set its opacity to 50%
+    labelWalletHDStatusIcon->show();
```

Comment:
Just a nit: shouldn't this line be above the comment?

Reply:
Indeed. Many thanks!